<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frame_tab_queue</class>
 <widget class="QFrame" name="frame_tab_queue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeWidget" name="tree_queue">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="headerDefaultSectionSize">
        <number>75</number>
       </attribute>
       <column>
        <property name="text">
         <string>Job Name</string>
        </property>
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Properties</string>
        </property>
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignLeft|AlignVCenter</set>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="btn_queue_remove">
         <property name="text">
          <string>Remove From Queue</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_queue_exe">
         <property name="text">
          <string>Execute Job</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_queue_exeall">
         <property name="text">
          <string>Execute All Jobs</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
